docs/proposals/super-simple-discovery-api.md

```diff
@@ -0,0 +1,306 @@
+  ],
+  "certificateAuthorities": [
+    "MIIDFDCCAfygAwIBAgIJAIr/AmnKEdesMA0GCSqGSIb3DQEBBQUAMBAxDjAMBgNV
+     BAMTBWpiZWRhMB4XDTE2MDgxMTIwMjc0MVoXDTQzMTIyODIwMjc0MVowEDEOMAwG
+     A1UEAxMFamJlZGEwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDAiCK6
+     t6LhHwojz+RQ9IVIOA2P+mL6XWSA1o6AaSMudzJRO2Gr60TFWHoz5i9+vp2ngZBR
+     NOwjIc1iykvV5rFP8mNZKB3xXo6XsKXUVDP4maiyOuvywkvtnhH/HU/5oIlVPDLz
+     2LWnOZd7Xip+Zgocd756p22J0oagmWien5OYHsWAlOqbut5UhfSH4gEdC/pJ+Qs8
+     6N292+2A3TzJC0fX9HrcCYJEQ1v2491p8RU+G3R+IFNXPrOvuY2dHN/w3BK+Vqet
+     tTKvINXCkSms6Nw9iqDjyWcRBhNg3l4j4yk5zZUmUQ2kQ0urXXZMaXSGSOZ/DLgy
+     fEAPbX5rUOL1SNeRAgMBAAGjcTBvMB0GA1UdDgQWBBRncoPpkKPwuBh5oWSEK+8o
+     Yo21wzBABgNVHSMEOTA3gBRncoPpkKPwuBh5oWSEK+8oYo21w6EUpBIwEDEOMAwG
+     A1UEAxMFamJlZGGCCQCK/wJpyhHXrDAMBgNVHRMEBTADAQH/MA0GCSqGSIb3DQEB
+     BQUAA4IBAQAZpAsBflGuYSgSCZujEpHRYmP+Pl/APe37+iBioi4JgMSMKydJFvFH
+     1mQaSJVc6yvNg/mqMF9drKbcX/fEdgx+25DqdXRovyvAV5/RlSRr0RA4xUaniQs0
+     H0D1VUo5H9/VCH4S5CwYYPPG/1uPyKxgsRWIm3oldNiE93NseSzd65JTwg7NMISg
+     9YMtbpdmDM8rrVm18QLjyjDdxEoKEfJtahl1cAw8XtoqYDczwMKITx4VoTE3gyMI
+     8TKIw3YNOzjzokvEzlsscMhWryIjARjRcJ+t9kztPxB6bUEid8aOoG3OpGjpjAlp
+     r4T3oIWo/YE8K2K1te/3gE/5L3QaKM2/",
+    "MIIDFDCCAfygAwIBAgIJAIr/AmnKEdesMA0GCSqGSIb3DQEBBQUAMBAxDjAMBgNV
+     BAMTBWpiZWRhMB4XDTE2MDgxMTIwMjc0MVoXDTQzMTIyODIwMjc0MVowEDEOMAwG
+     A1UEAxMFamJlZGEwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDAiCK6
+     t6LhHwojz+RQ9IVIOA2P+mL6XWSA1o6AaSMudzJRO2Gr60TFWHoz5i9+vp2ngZBR
+     NOwjIc1iykvV5rFP8mNZKB3xXo6XsKXUVDP4maiyOuvywkvtnhH/HU/5oIlVPDLz
+     2LWnOZd7Xip+Zgocd756p22J0oagmWien5OYHsWAlOqbut5UhfSH4gEdC/pJ+Qs8
+     6N292+2A3TzJC0fX9HrcCYJEQ1v2491p8RU+G3R+IFNXPrOvuY2dHN/w3BK+Vqet
+     tTKvINXCkSms6Nw9iqDjyWcRBhNg3l4j4yk5zZUmUQ2kQ0urXXZMaXSGSOZ/DLgy
+     fEAPbX5rUOL1SNeRAgMBAAGjcTBvMB0GA1UdDgQWBBRncoPpkKPwuBh5oWSEK+8o
+     Yo21wzBABgNVHSMEOTA3gBRncoPpkKPwuBh5oWSEK+8oYo21w6EUpBIwEDEOMAwG
+     A1UEAxMFamJlZGGCCQCK/wJpyhHXrDAMBgNVHRMEBTADAQH/MA0GCSqGSIb3DQEB
+     BQUAA4IBAQAZpAsBflGuYSgSCZujEpHRYmP+Pl/APe37+iBioi4JgMSMKydJFvFH
+     1mQaSJVc6yvNg/mqMF9drKbcX/fEdgx+25DqdXRovyvAV5/RlSRr0RA4xUaniQs0
+     H0D1VUo5H9/VCH4S5CwYYPPG/1uPyKxgsRWIm3oldNiE93NseSzd65JTwg7NMISg
+     9YMtbpdmDM8rrVm18QLjyjDdxEoKEfJtahl1cAw8XtoqYDczwMKITx4VoTE3gyMI
+     8TKIw3YNOzjzokvEzlsscMhWryIjARjRcJ+t9kztPxB6bUEid8aOoG3OpGjpjAlp
+     r4T3oIWo/YE8K2K1te/3gE/5L3QaKM2/"
+  ],
+  "fetchedTime": "2016-08-16T18:41:10+0000",
+  "expiredTime": "2016-08-16T21:41:10+0000"
+}
+```
+
+```go
+type ClusterInfo struct {
+  TypeMeta
+
+  // ClusterID is a globally unique (and not necessarily human readable) ID for a cluster.
+  ClusterID string
+
+  // Endpoints are base addresses to get to the Kubernetes API server.
+  Endpoints []string
+
+  // CertificateAuthorities is a list of root certificates to trust for
+  // connections to this cluster. These are PEM encoded certificates without
+  // `-----BEGIN CERTIFICATE-----` lines.
+  CertificateAuthorities [][]string
```

Comment:
In `Config` certs are represented as `[]byte`, which gets base64-encoded when marshaled, so we should probably stick to that, as we have machinery for it already.